app/code/Magento/Customer/view/adminhtml/templates/tab/view/sales.phtml

```diff
@@ -11,27 +11,27 @@ $singleStoreMode = $block->isSingleStoreMode();
 <div class="entry-edit fieldset-wrapper">
 
     <div class="fieldset-wrapper-title">
-        <span class="title"><?= $block->escapeHtml(__('Sales Statistics')) ?></span>
+        <span class="title"><?= $escaper->escapeHtml(__('Sales Statistics')) ?></span>
```

Comment:
It seems like escaper is not defined in this file

Reply:
fixed